lib/fake_idp/encryptor.rb

```diff
@@ -1,4 +1,5 @@
 require "xmlenc"
+require "builder"
```

Comment:
Why do we add builder here?

Reply:
Now I've read the commit message, I see that this was intentional, but I still don't understand _why_ the exception was being raise. How would we encounter this?

Reply:
I encountered this error when running specs:

```ruby
Failures:

  1) FakeIdp::SamlResponse encrypted assertion generates a valid SAML response
     Failure/Error: xml = Builder::XmlMarkup.new

     NameError:
       uninitialized constant FakeIdp::Encryptor::Builder
       Did you mean?  Bundler
     # ./lib/fake_idp/encryptor.rb:54:in `build_encryption_template'
     # ./lib/fake_idp/encryptor.rb:19:in `encrypt'
     # ./lib/fake_idp/saml_response.rb:72:in `encrypt_assertion!'
     # ./lib/fake_idp/saml_response.rb:58:in `build'
     # ./spec/saml_response_spec.rb:52:in `block (3 levels) in <top (required)>'
```

Similar to explicitly requiring `xmlenc`, which is in the standard library, we also need to require `builder` since they're not auto-loaded. For example,

```
❯ irb
irb(main):001:0> defined? Builder
=> nil
irb(main):002:0> require "builder"
=> true
irb(main):003:0> defined? Builder
=> "constant"
irb(main):004:0> defined? Xmlenc
=> nil
irb(main):005:0> require "xmlenc"
=> true
irb(main):006:0> defined? Xmlenc
=> "constant"
```

Reply:
Cool. Thanks for explaining.